### ISSUE TITLE
CHIEF pretrained features are worse than random initialization or simple mean pooling

### ISSUE THREAD

**Issue by amy-galon** [hms-dbmi/CHIEF#24]
We are following the concerns being raised about this study both publicly on this forum (#23, #20, #21), on pubpeer (https://pubpeer.com/publications/C8CFF9DB8F11A586CBF9BD53402001), and privately. Most concerns are around how the authors trained their model (#20, #21, #23) and downstream evaluation (#23). While many of these are serious and valid concerns, we wanted to assess the overall representation quality of CHIEF embeddings and its downstream performance. _**In our quantitative analysis on two separate datasets and classification tasks (binary and multiclass) we find that CHIEF slide embeddings are largely meaningless and even the mean of all patch features or random initialization performs better.**_ In our critique we will remain as objective as possible, back everything with explicit analysis and explain our experimental process in detail. We openly invite others in the community to conduct similar analysis and join forces in parsing this study. 

### CHIEF embeddings underperforms mean pooling and random pooling

To assess the representation quality of pretrained CHIEF embeddings, we compared with the following baselines:
**Mean Pooling:** Simply taking the average of all patch features in the bag and use it as the slide feature.
**Random Pooling:** Passing patch features through a randomly-initialized ABMIL model, with its output used as the slide feature. While the authors repeatedly argue that at the time of submission there were no other models available for comparison (https://github.com/hms-dbmi/CHIEF/issues/19#issuecomment-2345081445) **we believe that everyone would agree that mean pooling and random pooling are appropriate baselines and if CHIEF can't outperform the mean of all patch embeddings or random init. what is the purpose of the complex method proposed in the paper?** 

All pre-extracted slide features from CHIEF (Pretrain), mean pooling, and random pooling were evaluated via linear probe (logistic regression) following current practices in the self-supervised learning community (https://arxiv.org/abs/2304.07193). We evaluated two tasks: (1) TCGA-BRCA (5-fold site-stratified splits), and (2) EBRAINS (30-class brain tumor subtyping, using the same train-validation-test splits from UNI, these splits have been commonly used by many studies since the UNI authors released them). All comparisons used the same bags of pre-extracted CTransPath features (extracted at 10X via the CLAM pipeline, which was also used by the authors). We used the CTransPath model weights from this GitHub and not the old checkpoint. Overall, we report that slide pretraining in CHIEF is worse than mean pooling and random pooling.

|              |  | |              | TCGA-BRCA (binary) | ERAINS (30 class) |
| :---------------- | :------: | :----: | :----------------: | :------: | :----: |
| **ROI Encoder**        |**MIL Architecture**   |**Pretrain Method**| **Evaluation**| **AUROC** | **Bal. Accuracy** |
| CTransPath           |   ABMIL   | CHIEF              | Linear Probe | 0.905 | 0.542 |
| CTransPath    |  ABMIL   |Mean Pooling              | Linear Probe | 0.933 | 0.592 |
| CTransPath |  ABMIL   | Radom Pooling              | Linear Probe | 0.928 | 0.647 |

### CHIEF finetuning underperforms random initialization

To assess the effectiveness of finetuning capabilities of CHIEF, we also compared CHIEF with a randomly initialized ABMIL. We used the same hyper-parameters and code implementation as the repository. We used the same tasks and splits reported above. We report that CHIEF finetuning also underperforms random initialization.

|              |  | |              | TCGA-BRCA (binary) | ERAINS (30 class) |
| :---------------- | :------: | :----: | :----------------: | :------: | :----: |
| **ROI Encoder**        |**MIL Architecture**   |**Pretrain Method**| **Evaluation**| **AUROC** | **Bal. Accuracy** |
| CTransPath |  ABMIL   |CHIEF              | Finetune | 0.884 | 0.549 |
| CTransPath |  ABMIL   | Random Init.              | Finetune | 0.913 | 0.567 |

### Comment

Give that the authors already published CTransPath (Medical Image Analysis) as an ROI level encoder, if CHIEF slide encoder can’t outperform the mean of all patch embeddings or random init., it heavily undermines and negates the value of the pretraining paradigm proposed in the study, given that it required 60k slides with corresponding labels. We invite the authors @Xiyue-Wang @Dadatata-JZ @khyu and anyone else to verify our results feel free to  ask questions. In particular it would be great to hear from the senior author @khyu if they still stand by their approach and study in light of this analysis and previous concerns raised by the community. 

### Comment to the community

We are aware of multiple efforts to conduct similar analyses, in our opinion publicly posting concerns and results allows the authors to respond and the community to vet our analysis rather than direct and immediate communication with nature editors, rest assured at this point they are already aware and monitoring the situation. If anything, we propose joining forces to compile concerns, conduct a thorough and fair analysis while giving the authors the opportunity to respond. In the coming days we plan to continue this analysis, compile all concerns and post publicly before going back to the EIC.

**Comment by Dadatata-JZ** [comment]
@amy-galon 

Hi Amy, 
Thanks for sharing this. I quickly went over your type-up posted last night. The results are quite interesting. Here are some of my initial thoughts, and we can continue the discussion as needed.

First and foremost, while these two tasks may have been evaluated elsewhere, neither is included in CHIEF's presentation. 

Some implementation details also don’t fully align, and since we don’t use logistic regression for classification tasks in CHIEF, I’m afraid that I can't comment on that aspect. We are quite familiar with EBrains and its sample distributions;  implementation variations or perturbations may need to be noted. Personally, I feel CNS tumors are particularly complex and require substantial efforts from all of us to investigate more ;)

That said, these shouldn’t discourage you and us from potential joint exploration. We’d be happy to discuss your experimental setups for these extended validations if you’d like to post your pipe somewhere.

As always, we’re open to discuss in depth and providing support, even during off-hours for maintenance. Drop us an email. 

Cheers

**Comment by Dadatata-JZ** [comment]
Amy, @amy-galon 

Thank you for your enthusiasm. Due to time differences, we may not be able to catch up promptly, but surely do our best!

These numbers look interesting, for instance, your numbers from simple tests seem already higher than the published knowledge. IMO, you may consider publishing this somewhere (e.g., IDH prediction, you easily crashed 0.91 compared to other existing publications)! 

However, since the numbers are not aligned with our experiments, to navigate this evaluation better, feel free to post your codes, experimental setup, data processing and partitions online. We can discuss from there with more context. 

Have a good night.  
Cheers,
